tests/rpc.rs

```diff
@@ -532,3 +511,17 @@ async fn account_call() -> Result<(), Error> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn deploy_new_account_on_sepolia() -> Result<(), Error> {
+    // TODO #807
```

Comment:
I agree. I have thought it through and came to three obstacles :

1. Make changes within template contract so that new account is deployed - simple id update
2. Commit changes within github workflows to have automated test

Last thing that comes to my mind, is that with every new account we get `key.json` and `account.json` files which are needed for `declare` and `deploy` on Sepolia network. This gives two options which I need help with decision:

1. We can easily delete them when test ends and leave accounts in the deserts of the blockchain without the ability to access them
2. We can store all these values in some additional folder so that we can access them later

I guess that storing them would be a better way and committing them in some folder, but having in mind that `key.json` and `account.json` might not be the best things to have publicly exposed.